lib/peer-discovery.js

```diff
@@ -117,7 +118,7 @@ module.exports = class PeerDiscovery {
     const server = this._serverSessions > 0
     const client = this._clientSessions > 0
 
-    if (this.destroyed) return
+    if (this.destroyed || this.suspended) return
```

Comment:
Not related to this PR, but seems like this.destroyed can never be true here due to line 116?

Reply:
Good point!